Wikipedia/Categories/UIImage+WMFStyle.m

```diff
@@ -14,4 +14,15 @@ + (UIImage*)wmf_placeholderImage {
     return img;
 }
 
++ (instancetype)wmf_imageFromColor:(UIColor*)color {
+    CGRect rect = CGRectMake(0, 0, 1, 1);
```

Comment:
ah, i was thinking we were using this on some scrolling/reused view. if it's only done once for all 3 buttons at `viewDidLoad` it should be OK for now